app/views/person_mailer/welcome_email.html.erb

```diff
@@ -24,7 +27,8 @@ Set Up Your Gmail Messages To Be From:  <%= @person.email %>
 </b>
 <ol>
   <li>
-    Log into your CMU-SV Gmail account at <a href="http://gmail.sv.cmu.edu/">http://gmail.sv.cmu.edu/</a>.
+    Log into your CMU-SV Gmail account at <a href="http://gmail.sv.cmu.edu/">http://gmail.sv.cmu.edu/</a>
+(a <a href="https://mail.google.com/a/west.cmu.edu/">re-direct</a>).
```

Comment:
Added the second URL because the first was actually broken for some time (months, at least, fixed just a couple days ago).  Will think about it when tweaking the page (for your first comment).